src/main/resources/db/changelog/v1.0.0/11-create-testData-table-product.xml

```diff
@@ -12,7 +12,7 @@
             <column name="name" value="Yellow core wood table"/>
             <column name="category" value="table"/>
             <column name="description" value="Lorem Ipsum comes from a latin text written in 45BC by Marcus Tullius Cicero."/>
-            <column name="photo_link" value="https://thumbs2.imgbox.com/04/f6/jTjRb0G3_t.png"/>
+            <!--  <column name="photo_link" value="https://thumbs2.imgbox.com/04/f6/jTjRb0G3_t.png"/>-->
```

Comment:
i think we can remove these lines. doesn't make sense to keep the lines commented